pkg/listener/kafka/kafka.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
```

Comment:
I would take the opportunity to split all the Kafka manifests into a different jsonnet so everything related to kafka is self contained. Also we would remove the mandatory deployment of Kafka (see [this user feedback](https://bitnami-oss.slack.com/archives/C5A89A32T/p1516355853000153)).

Reply:
I made it in this separate PR https://github.com/kubeless/kubeless/pull/570